content/agent/basic_agent_usage/windows.md

```diff
@@ -70,10 +70,30 @@ You can also use Windows Powershell if you are running on a modern version of Wi
 The Agent can be started, stopped, and restarted from the Services panel. To view the Services panel, execute the following in a `cmd.exe` shell: `services.msc`. Once you're in the console, find the "Datadog Agent" service. Right clicking on the service reveals options to start, stop, and restart the Agent.
 
 ## Status and Information
+##### Agent >= 6
 
-To check if the Agent is running, check if the service status in the Services panel is listed as "Started". A process called "ddagent.exe" should also exist in the Task Manager. To receive more information about the Agent's state, visit the _status page_ by going to **Settings -> Agent Status** in Agent version 5.2 and above and by going to `http://localhost:17125/status` in Agent version 3.9.1 to 5.1.
+To check if the Agent is running, check if the `DatadogAgent` service in the Services panel is listed as *Started*. A process called *Datadog Metrics Agent* (`agent.exe`) should also exist in the Task Manager.
 
-For 5.2 and later versions of the Agent go to the Datadog Agent Manager->Settings->Agent Status
+To receive more information about the Agent's state, start the Agent GUI by either:
+
+- Right clicking on the Datadog Agent system tray icon -> Configure
+- running `& 'C:\program files\datadog\datadog agent\embedded\agent.exe' launch-gui` from an admin Powershell prompt
```

Comment:
Are the above two sequential steps, or are they two different options? I'm confused by the word "either" in line 77.